### PR TITLE
Implement adding arrow object

### PR DESCRIPTION
**Pull request by Yun-Jung**
This file is fed with Logging Server and creates the arrow object by credential provided by json file.

### CODE REVIEW

File: R/faasr_arrow_s3_bucket.R [hunk 1]
```diff
@@ -0,0 +1,27 @@
+library("arrow")
+
+
+faasr_arrow_s3_bucket <- function(server_name=.faasr$LoggingServer) {
```
Comment (spark0510) [inline R/faasr_arrow_s3_bucket.R:4]: Set no default value for the argument "server_name"
`server_name=.faasr$LoggingServer` -> `server_name`

**Review by spark0510 — CHANGES_REQUESTED** [review]

**Comment by renatof** [comment]
Thanks folks. Sorry for the change of mind, but now I am leaning towards the original goal of allowing using a default server. I guess my main issue is that we're effectively electing the LoggingServer as default, and that may not be intuitive to the user that "Logging" means "Default"
One possibility to make that more clear and explicit would be to change the JSON key "LoggingServer" to "DefaultDataServer" to make it more explicit, and then allow that to be the default for S3 put/get and arrow. We can discuss that before making a change

**Comment by renatof** [comment]
actually, DefaultDataStore to be consistent with DataStores

**Comment by renatof** [comment]
closing as it this been addressed in a different pull request. thanks Yun-Jung, Sungjae!

**Pull request closed without merging**